google-cloud-storage/src/test/java/com/google/cloud/storage/spi/v1/HttpStorageRpcTest.java

```diff
@@ -51,13 +51,122 @@
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 
+class TestRequest extends LowLevelHttpRequest {
```

Comment:
I meant these should be in new files.

Reply:
Are you sure? These classes unlikely will be used by anyone else but HttpStorageRpcTest. 
I can move them to separate files if you insist, but IMHO it will only make understanding harder. 
If it is just a matter of taste, I would prefer to see them as inner classes.

Reply:
Not sure if there's an explicit style rule but Google almost never uses non-public outer classes in the same .java file. 

In general inner classes encourage violation of data encapsulation and the single responsibility principle. 

And perhaps most importantly here, the sheer size of a file becomes a problem. I found this harder to read as one large class than as several smaller classes.

Reply:
Thanks for the explanation. I'll move the classes with code to new files and, if you don't mind, keep the trivial classes  (MockData and RpcRequestHolder) as inner.

Reply:
All inner classes moved to new files